src/resources/assets/js/geo/components/filterListComponent.vue

```diff
@@ -0,0 +1,44 @@
+<template>
+    <span>
+        <strong>with<span v-if="rule.negate">out</span></strong>
+        <span v-text="name"></span>
+        <strong v-if="dataName" v-text="dataName"></strong>
+    </span>
+</template>
+
+<script>
+/**
+ * Component for a geo-filter list item
+ *
+ * @type {Object}
+ */
+export default {
+    props: {
+        rule: {
+            type: Object,
+            required: true,
+        },
+    },
+    data() {
+        return {
+            name: this.rule.id,
+        };
+    },
+    methods: {
+        createName(ids) {
+            let typeForm = ids.length === 1 ? " image" : " images";
+            return "".concat("(", ids.length, typeForm, ")");
```

Comment:
You can also use template literals in JS:

```suggestion
            return `(${ids.length} ${typeForm})`;
```

Reply:
resolved fe71ba927e5c9139f8079b9d3351476a121bd2be